src/Avalonia.Visuals/Media/Transformation/TransformParser.cs

```diff
@@ -0,0 +1,444 @@
+using System;
+using System.Globalization;
+using Avalonia.Utilities;
+
+namespace Avalonia.Media.Transformation
+{
+    public static class TransformParser
+    {
+        private static readonly (string, TransformFunction)[] s_functionMapping =
+        {
+            ("translate", TransformFunction.Translate),
```

Comment:
wouldnt it be more efficient to use a dictionary for these?

Reply:
It would be if I was using strings to look it up. I am using spans so dictionary does not help right now (maybe one day they will introduce better support for ROS<char>)